vscode/src/projectSystem.ts

```diff
@@ -104,9 +70,31 @@ async function findManifestDocument(
   return null;
 }
 
+async function tryReadManifestInDir(
+  uriToQuery: URI,
+): Promise<{ uri: vscode.Uri; content: string } | null> {
+  const potentialManifestLocation = Utils.joinPath(uriToQuery, "qsharp.json");
+
+  let listing: { uri: vscode.Uri; content: string } | null = null;
+  try {
+    listing = await readFileUri(potentialManifestLocation);
+  } catch (err) {
+    log.error("Error thrown when reading file: ", err);
+  }
+  return listing;
+}
+
```

Comment:
Could this just become an exists check of some kind, rather than reading the contents?

Reply:
Simplified to not read contents.